### PR TITLE
Feature/add api token command key support #3

### PR DESCRIPTION
**Pull request by HMarzban**
Hi, as I mentioned in issue #3, I found a way to make it easy deployment for CI/CD.
now we can use `cli` like below:

```bash
$ liara deploy --port <project-port> --project <project-name> --api_token <your-token>
```
If you have any further questions about the concept of implementations, I'll be glad to be accountable 😊

### CODE REVIEW

**Comment by mhemrg** [comment]
Wow. Great PR :blush: :+1: 
I

File: src/liara.js [hunk 1]
```diff
@@ -33,7 +33,12 @@ const liaraConfPath = join(homedir(), '.liara.json');
 
 let liaraConf;
 try {
-  liaraConf = JSON.parse(readFileSync(liaraConfPath));
+  if(!args.api_token){
+    liaraConf = JSON.parse(readFileSync(liaraConfPath));   
+  }else{
+    liaraConf = {}
+    liaraConf["api_token"] = args.api_token;
```
Comment (mhemrg) [inline src/liara.js]: Using underline in the command line is not a good convention.
I recommend using `--api-token`.

**Comment by mhemrg** [comment]
Thank you @HMarzban. Merged. :tada:

**Comment by HMarzban** [comment]
It was my pleasure 😊

**Comment by mhemrg** [comment]
I didn't test this PR myself and seems that it doesn't work properly.

File: src/liara.js [hunk 2]
```diff
@@ -33,7 +33,12 @@ const liaraConfPath = join(homedir(), '.liara.json');
 
 let liaraConf;
 try {
-  liaraConf = JSON.parse(readFileSync(liaraConfPath));
+  if(!args.api_token){
```
Comment (mhemrg) [inline src/liara.js]: Still uses `api_token` with the underline.

File: package.json [hunk 3]
```diff
@@ -16,13 +16,14 @@
   "author": "Mhe Mrg <mhemrg120@gmail.com> (http://liara.ir/)",
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-backpack": "^0.4.3",
+    "@babel/core": "^7.4.3",
```
Comment (mhemrg) [inline package.json]: Please don't upgrade npm packages unless you're sure that the project is compatible with them.
Currently, the CLI doesn't work with these new versions.

File: src/liara.js [hunk 4]
```diff
@@ -33,7 +33,12 @@ const liaraConfPath = join(homedir(), '.liara.json');
...
+    liaraConf = JSON.parse(readFileSync(liaraConfPath));   
+  }else{
+    liaraConf = {}
+    liaraConf["api-token"] = args.api_token;
```
Comment (mhemrg) [inline src/liara.js]: I think this would be correct:
```javascript
liaraConf.api_token = args["api-token"];
```

**Comment by mhemrg** [comment]
I fixed the issue myself :)

File: src/liara.js (hunk 2 above)
Comment (HMarzban) [reply src/liara.js]: `api_token` is default property we get from `liaraConf` variable when user login successfully.

File: package.json (hunk 3 above)
Comment (HMarzban) [reply package.json]: with  the last config of `bable-core` which was version 6 is not compatible with `babel-loader` and I face to an error a lot,
now it's compatible with new version of `babel-core`, and I ran test suite and it was okay.

File: src/liara.js (hunk 4 above)
Comment (HMarzban) [reply src/liara.js]: you right

**Comment by HMarzban** [comment]
Let me correct all these issues and write a new unit test for this **PR**. then I let you know to check it up all features again.

**Pull request merged**